cnf-tests/pod-utils/cyclictest-runner/main.go

```diff
@@ -29,10 +28,6 @@ func main() {
 		klog.Fatalf("failed to get self allowed CPUs: %v", err)
 	}
 
-	if *threads == -1 {
```

Comment:
This is not needed anymore. The number of CPUs will be set by the user using the new variable and we'll always run the tool on every CPU which was allocated under the pod.

Reply:
this is great!